include/boost/fusion/container/vector/detail/config.hpp

```diff
@@ -15,6 +15,8 @@
   || defined(BOOST_NO_CXX11_RVALUE_REFERENCES) \
   || defined(BOOST_NO_CXX11_TEMPLATE_ALIASES)  \
   || defined(BOOST_NO_CXX11_DECLTYPE)) \
+  || defined(BOOST_NO_CXX11_FUNCTION_TEMPLATE_DEFAULT_ARGS) \
```

Comment:
Sorry for late reply.

> Yes, but should tuple check this separately since it also requires it? I only bring it up because my next patch will add the `NO_CXX11_FUNCTION_TEMPLATE_DEFAULT_ARGS` requirement to `fusion::vector` too.

I think the check is unnecessary, but acceptable. I worry about "too much" or "insufficient".